### PR TITLE
#25 - Sponsor page use yml

### PR DESCRIPTION
**Pull request by jcklie**
- Use new real data for sponsor page
- Add images

Needs https://github.com/acl-org/acl-2020-virtual-conference-sitedata/pull/10

I did not change the layout yet.

### CODE REVIEW

**Comment by jcklie** [comment]
The session timestamps are not localizing yet, I tried the same solution as on the https://github.com/acl-org/acl-2020-virtual-conference/blob/master/templates/poster.html but I did not get it working yet.

**Comment by GuyAglionby** [comment]
The [regex](https://github.com/acl-org/acl-2020-virtual-conference/blob/a0982cbd6930deedff415365658fcc64def01ccc/static/js/time-extend.js#L2) to match times is fussy and wants it to be surrounded by parentheses. This should probably be relaxed - the poster page things can be easily changed to allow this ([here](https://github.com/acl-org/acl-2020-virtual-conference/blob/a0982cbd6930deedff415365658fcc64def01ccc/templates/poster.html#L73), [here](https://github.com/acl-org/acl-2020-virtual-conference/blob/a0982cbd6930deedff415365658fcc64def01ccc/main.py#L61))

File: static/images/sponsors/foo.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
+https://acl2020.org/assets/images/logos/Bloomberg-logo-2019.png
```
Comment (hao-fang) [inline static/images/sponsors/foo.txt:1]: do we need this `foo.txt` file?

**Comment by jcklie** [comment]
I fixed the times, update the submodule and deleted the bogus files. 

image

Are we gonna keep this font and blue header color? For me, it does not look ACL like.

**Comment by jcklie** [comment]
Can you help me with the submodule and why there is a conflict? I am not too familiar with submodules.

**Comment by hao-fang** [comment]
> Can you help me with the submodule and why there is a conflict? I am not too familiar with submodules.

Since the PRs on the `sitedata_acl2020` are merged, you can do
```bash
cd sitedata_acl2020
git pull
cd ..
git commit -am "bump sitedata_acl2020 version"
```
This should work. But let me know if doesn't -- we can chat on RocketChat.

**Pull request merged**